tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -7279,6 +7279,8 @@ gen_arg_operand(LL_ABI_Info *abi, unsigned abi_arg, int arg_ili)
     operand = gen_llvm_expr(value_ili, arg_type);
   if (arg->kind == LL_ARG_BYVAL && !missing)
     operand->flags |= OPF_SRARG_TYPE;
+  if (arg->kind == LL_ARG_INDIRECT && !missing && (abi_arg == 0))
```

Comment:
IMO it would be easier to read if it could have more parentheses...

Reply:
Heh, I feel the opposite. `abi_arg == 0` in a compound `&&` expression doesn't really need the parentheses. I can change it if you insist.